webpack/assets/javascripts/react_app/routes/index.js

```diff
@@ -47,7 +36,7 @@ const AppSwitcher = () => {
     const nextPath = getURIPath();
     if (currentPath !== nextPath) {
```

Comment:
This is now causing an endless loop, since `currentPath` is reset to null on page load

Reply:
pushed a commit that should fix this